task_processor/migrations/helpers/__init__.py

```diff
@@ -0,0 +1,29 @@
+import os
+import typing
+from contextlib import suppress
+
+from django.db import migrations
+
+
+class PostgresOnlyRunSQL(migrations.RunSQL):
```

Comment:
Why was this file created in `__init__.py` instead of a normally named python module?

Reply:
Ugh, yeah, sorry - I should have responded to this at the time when it was fresh in my memory but I basically had endless pain here. For some reason, when this was defined in just `helpers.py`, I was getting spurious errors that I couldn't explain. I'm going to retry it now and will either update here that it's fixed, or add a comment to this file explaining why I can't fix it.

Reply:
I've refactored this slightly.

Reply:
This may also be somewhat related to the pain [here](https://github.com/Flagsmith/flagsmith-workflows/pull/24/commits/957c418f514f0296c370ad1977f47be47d120732).

Reply:
Ouch ok I'll leave it to you when to cut your losses on this.

Reply:
I did eventually work out what the issue was here. It was the fact that the `pyproject.toml` file didn't have the build system defined which meant that non-python files weren't being included in the locally built package after cloning the git dependency.